src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -165,40 +177,41 @@ def execute(self, **kwargs):
 
 command_table = CommandTable()
 
-def get_command_table(module_name=None):
+# Map to determine with module a command was registered in
+command_module_map = {}
+
+def load_params(command):
+    module_to_load = command_module_map.get(command, None)
+    if not module_to_load:
+        return
+    command_table[command].load_arguments()
+    import_module('azure.cli.command_modules.' + module_to_load).load_params(command)
+    _update_command_definitions(command_table)
+
+def get_command_table():
     '''Loads command table(s)
     When `module_name` is specified, only commands from that module will be loaded.
     If the module is not found, all commands are loaded.
     '''
-    loaded = False
-    if module_name:
+    installed_command_modules = []
+    try:
+        mods_ns_pkg = import_module('azure.cli.command_modules')
+        installed_command_modules = [modname for _, modname, _ in \
+                                    pkgutil.iter_modules(mods_ns_pkg.__path__)]
+    except ImportError:
+        pass
+    logger.info('Installed command modules %s', installed_command_modules)
+    logger.info('Loading command tables from all installed modules.')
```

Comment:
I guess it doesn't matter since we load all command modules by default. This may be vestigial code from when we tried the "load one module" optimization.